WFD/RBD.py

```diff
@@ -154,67 +156,60 @@ def create_new_rbd_item(self, entry_data, country):
         :param entry_data: dict with the data for the rbd
         :param country: country code
         """
-        data = {
-            'labels': {},
-            'descriptions': {}}
-
-        data['labels'] = self.make_labels(entry_data)
-        data['descriptions'] = self.make_descriptions(entry_data, country)
+        labels = WfdBot.convert_language_dict_to_json(
```

Comment:
It was used to set the country part of the description. This is now instead handled by the internal `self.country` (in `WfdBot`).

But I realised I need `rbd.set_common_values(data)` in order to set this.

Fixed